From bde1e3a7c9804d4bd1e4d2361360e4621386acd5 Mon Sep 17 00:00:00 2001
From: Keith Packard <keithp@keithp.com>
Date: Wed, 18 Feb 2009 16:43:23 -0800
Subject: [PATCH] X11/fonts/fontmod.h doesn't exist anymore.

libXfont 1.4.0 no longer includes this file, nor do we need it, so just stop
attempting to reference it.

Signed-off-by: Keith Packard <keithp@keithp.com>
(cherry picked from commit 4557b3f6c4273cd83b701beaf7a150c806fed298)
---
 hw/xfree86/loader/loaderProcs.h |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/hw/xfree86/loader/loaderProcs.h b/hw/xfree86/loader/loaderProcs.h
index b71ad45..85c9109 100644
--- hw/xfree86/loader/loaderProcs.h
+++ hw/xfree86/loader/loaderProcs.h
@@ -56,7 +56,6 @@
 #undef IN_LOADER
 #define IN_LOADER
 #include "xf86Module.h"
-#include <X11/fonts/fontmod.h>
 
 typedef struct module_desc {
     struct module_desc *child;
-- 
1.6.1.2

